### PR TITLE
Update tests.yml

### PR DESCRIPTION
**Pull request by imbsky**
- `[]` is not required.
- It's dangerous to use master, so please specify a tag for any Action.

### CODE REVIEW

**Comment by imbsky** [comment]
Updated.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
     - name: Install RubyGems
       run: |
         gem install --user bundler
-        bundle install --retry=3 --jobs=4
+        bundle install --retry=3 --jobs=$(getconf _NPROCESSORS_ONLN)
```
Comment (imbsky) [inline .github/workflows/tests.yml]: Actions VMs are not that powerful, so 4 is too many. So, specify the number of processors on the machine. This allows the same code to be used in the future even if the Actions VM becomes more powerful.

**Comment by MikeMcQuaid** [comment]
I don't see the need for any of these changes. As-is they are consistent with the other Homebrew organisation repos, too.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (EricFromCanada) [reply .github/workflows/tests.yml]: Homebrew uses its own worker nodes.

Comment (MikeMcQuaid) [reply .github/workflows/tests.yml]: Not for this repo, just FYI.

**Comment by MikeMcQuaid** [comment]
Passing on this for now, sorry!

**Pull request closed without merging**